### PR TITLE
Added Datadog Downtime module

### PR DESCRIPTION
**Pull request by dzinek**
##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

datadog_downtime - Manages Datadog downtimes
##### ANSIBLE VERSION

```
ansible 2.3.0 (devel b122775937) last updated 2017/01/11 21:54:30 (GMT +200)
```

##### SUMMARY

I've added another module for Datadog - Scheduled Downtime.
I've based on datadog_monitor, to still have similar structure of DD modules.

This feature will add a possibility to schedule, update and cancel Downtimes.

### CODE REVIEW

**Comment by hush-hush** [comment]
Hi @dzinek,

I'm from the Datadog team and it looks good to me.

Thank you for the contribution !

File: lib/ansible/modules/monitoring/datadog_downtime.py [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+def schedule_downtime(module):
+    downtime = _get_downtime(module)
+    if not downtime:
+        _post_downtime(module)
+    else:
+        _update_downtime(module, downtime)
+
+def cancel_downtime(module):
+    downtime = _get_downtime(module)
+    if not downtime:
```
Comment (ghost) [inline lib/ansible/modules/monitoring/datadog_downtime.py:195]: This doesn't allow the caller to cancel downtime by scope. _get_downtime returns an empty dictionary when id is not specified. It'd be great if this was changed to require id or scope instead of just id.

**Comment by Akasurde** [comment]
@dzinek Are you still working on this ?

**Comment by alexcurtin** [comment]
Can this be merged into core? If not can I test against Ansible 2.7 and open the same PR to get this merged? @Akasurde

**Comment by Akasurde** [comment]
@alexcurtin Yes, I think original author is not responding. You can go ahead and test against 2.7 and let me know. I will help.

File: .gitignore [hunk 2]
```diff
@@ -87,3 +87,4 @@ htmlcov/
 .coverage
 # ansible-test coverage results
 test/units/.coverage.*
+.venv/
```
Comment (mattclay) [inline .gitignore:90]: This change should be made in a separate PR since it's not directly related to the rest of the changes here.

**Comment by Akasurde** [comment]
@jdoss There are pending reviews for this PR, once those are resolved we can merge this. 

@dzinek Are you still working on this issue ? Thanks.

**Comment by gundalow** [comment]
Could you please rebase and move this to the `/datadog` sub directory, thanks https://github.com/ansible/ansible/pull/64502

**Comment by Akasurde** [comment]
Closing this due to inactivity from contributor. Please feel free to open a new PR for this module. Thanks for the contribution.
